packages/override-tools/src/Cli.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as FileRepository from './FileRepository';
+import * as Manifest from './Manifest';
+
+import * as chalk from 'chalk';
+import * as fs from 'fs';
+import * as ora from 'ora';
+import * as os from 'os';
+import * as path from 'path';
+import * as yargs from 'yargs';
+
+import CrossProcessLock from './CrossProcessLock';
+import {GitReactFileRepository} from './GitReactFileRepository';
+import {OverrideFileRepositoryImpl} from './OverrideFileRepository';
+
+doMain(() => {
+  yargs
+    .command(
+      'validate <manifest> [version]',
+      'Verify that overrides are accurate and up-to-date',
+      cmdYargs =>
+        cmdYargs.options({
+          manifest: {type: 'string', describe: 'Path to an override manifest'},
+          version: {type: 'string', describe: 'React Native Version'},
+        }),
+      cmdArgv => validateManifest(cmdArgv.manifest, cmdArgv.version),
+    )
+    .epilogue(
+      'This tool allows maging JavaScript overrides for React Native Windows',
+    )
+    .demandCommand()
+    .recommendCommands()
+    .strict()
+    .showHelpOnFail(false)
+    .wrap(yargs.terminalWidth())
+    .version(false).argv;
+});
+
+/**
+ * Check that the manifest correctly describes files and is up to date
+ */
+async function validateManifest(manifestPath: string, version?: string) {
```

Comment:
>version? [](start = 54, length = 8)

Why would we want this to be optional?

Reply:
The version? By default we will check validity against the installed version. Optionally using a separate version let's you choose a different one to test against (E.g. during an upgrade)